docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,891 @@
+
+.. note:: Process costing methods are available for most of the unit operations in the IDAES Unit Model Library (Compressor, CSTR, Flash, Heater, HeatExchanger, HeatExchangerNTU, PFR, PressureChanger, Pump, StoichiometricReactor, Turbine, and generic vessels).
+
+The process costing module outlines capital cost methodology for generic IDAES unit operations assuming a 2018 cost basis.
+
+Table 1. Main Variables added to the unit block ("self.costing").
+
+=========================== ============================== ======== ==========================================================================
+Variable                    Symbol                         Units    Notes
+=========================== ============================== ======== ==========================================================================
+Purchase cost               :math:`purchase\_cost`         dollars  Purchase cost
+Base cost per unit          :math:`base\_cost\_per\_unit`  unitless Base cost per unit
+Base cost                   :math:`base\_cost`             unitless Base cost (base cost per unit * number of units)
+Number of units             :math:`number\_of\_units`      unitless Number of units to be costed (to take advantage of the economics of scale)
+=========================== ============================== ======== ==========================================================================
+
+.. note:: number of units by default is fixed to 1 and the user must unfix this variable to optimize the number of units. Also, `number of units` can be built as a continuous variable or an integer variable. If the latter, the user must provide an mip solver. Use the global costing argument for this purpose (integer_n_units=True or False).
+
+Available IDAES Process Costing Module Methods
+----------------------------------------------
+
+A default costing module has been developed primarily based on base cost and purchase cost correlations 
+from the following reference with some exceptions (noted in the documentation as appropiate).
+
+Process and Product Design Principles: Synthesis, Analysis, and Evaluation. Seider, Seader, Lewin, Windagdo, 3rd Ed. John Wiley and Sons. Chapter 22. Cost Accounting and Capital Cost Estimation
+
+Users should refer to the reference above for details of the costing correlations, however, a summary of this methods is provided below.
+
+The base costing framework provides a method to automatically register currency units for the costing block based on Chemical Engineering (CE) Cost Index conversion rates for US Dollars. IDAES will check is Pyomo has standard currency units registered, and if not will load unit conversions between USD at CE indices of 500 and 394, as well as USD annually from 1990 to 2020.
+    
+Table 2. Cost basis for each unit model.
+
+=========================== =========================  ==============
+Unit Model                  Basis                      Units
```

Comment:
I would say "Basis Units" or something to that extent to try to make it clearer what is meant (e.g. heat exchangers are costed based on area in feet, not that the cost is given in feet).